docs/documentation/getting-started.md

```diff
@@ -35,8 +35,7 @@ You need an **XMPP account** to use Saros.
     For example your Google, GMX, or web.de mail address. **They are
     valid XMPP accounts**.
     To use your Google account you have to allow less secure apps access
-    your account, see
-    <https://support.google.com/accounts/answer/6010255?hl=en>
+    your account, see [here](https://support.google.com/accounts/answer/6010255?hl=en).
     Unfortunately, gmx and web.de accounts do not work currently.
```

Comment:
Is it really useful to mention gmx and web.de specifically in line 35 if they don't work anyway?